decidim-elections/app/cells/decidim/votings/content_blocks/highlighted_votings_cell.rb

```diff
@@ -3,39 +3,30 @@
 module Decidim
   module Votings
     module ContentBlocks
-      class HighlightedVotingsCell < Decidim::ViewModel
-        delegate :current_user, to: :controller
-
-        def show
-          render if highlighted_votings.any?
-        end
+      class HighlightedVotingsCell < Decidim::ContentBlocks::HighlightedParticipatorySpacesCell
+        BLOCK_ID = "highlighted-votings"
 
-        def max_results
-          model.settings.max_results
-        end
+        delegate :current_user, to: :controller
 
-        def highlighted_votings
-          OrganizationPrioritizedVotings.new(current_organization, current_user)
-                                        .query
-                                        .limit(max_results)
+        def highlighted_spaces
+          @highlighted_spaces ||= OrganizationPrioritizedVotings.new(current_organization, current_user).query
         end
 
         def i18n_scope
           "decidim.votings.pages.home.highlighted_votings"
         end
 
-        def decidim_votings
-          Decidim::Votings::Engine.routes.url_helpers
+        def all_path
+          Decidim::Votings::Engine.routes.url_helpers.votings_path
+        end
+
+        def max_results
+          model.settings.max_results
         end
 
         private
 
-        def cache_hash
-          hash = []
-          hash.push(I18n.locale)
-          hash.push(highlighted_votings.map(&:cache_key_with_version))
-          hash.join(Decidim.cache_key_separator)
-        end
+        def block_id = BLOCK_ID
```

Comment:
Same comment as in [highlighted_assemblies_cell.rb](https://github.com/decidim/decidim/pull/10920/files#diff-b3db5c57a4634c36562d7145ffcab03aebd4955fa796a9daa5b031fac9bcdb9e)